ext/droption/droption.h

```diff
@@ -584,9 +584,7 @@ droption_t<bytesize_t>::convert_from_string(const std::string s)
     std::string toparse = s;
     if (scale > 1)
         toparse = s.substr(0, s.size() - 1); // s.pop_back() only in C++11
-    // While the overall size is likely too large to be represented
-    // by a 32-bit integer, the prefix number is usually not.
-    int input = atoi(toparse.c_str());
+    long long input = atoll(toparse.c_str());
```

Comment:
Thank you for the pull request!

Could you add a test of a large number passed to a bytesize_t option to the suite/tests/client-interface/option_parse.dll.cpp test?  (Run "git grep val_sep2" to see which files are involved.)

Reply:
Sorry for the delay.  Please let me know if I didn't add the test correctly.